grakn-test-tools/src/main/java/ai/grakn/test/GraphContext.java

```diff
@@ -77,7 +77,7 @@ public GraphContext assumeTrue(boolean bool){
     }
 
     public static void loadFromFile(GraknGraph graph, String file) {
-        GraphLoader.loadFromFile(graph, "../grakn-test-tools/src/main/graql/" + file);
+        GraphLoader.loadFromFile(graph, System.getProperty("main.basedir")+"/grakn-test-tools/src/main/graql/" + file);
```

Comment:
We have `GraknSystemProperty` for this sort of thing. Just add another enum there 👍